src/app/static/src/app/store/adr/actions.ts

```diff
@@ -59,6 +60,13 @@ export const actions: ActionTree<ADRState, RootState> & ADRActions = {
             });
     },
 
+    async getReleases(context, selectedDatasetId) {
+        await api<ADRMutation, ADRMutation>(context)
+            .withError(ADRMutation.SetADRError)
```

Comment:
Need to be careful here: ADR errors are only displayed in the modal... which is hidden as soon as you click Import. So they are effectively invisible. Could either write to baseline errors or attempt to keep dialog open until dataset details are retrieved (latter sounds more complicated).

Reply:
..or show ADR errors in baseline component, though that show more than we want if it includes all possible ADR errors